<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GADemoShowWidget</class>
 <widget class="QWidget" name="GADemoShowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,100,100">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>设置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="hideProPlotCk">
        <property name="text">
         <string>隐藏问题图像</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="onlyShowResCk">
        <property name="text">
         <string>全屏显示图像</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="hideAverResCk">
        <property name="text">
         <string>隐藏平均情况</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearResBtn">
        <property name="text">
         <string>清空迭代图像</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearAll">
        <property name="text">
         <string>清空所有</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCustomPlot" name="Pro_CustomPlot" native="true">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCustomPlot" name="Res_CustomPlot" native="true">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="bottomShowWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="OutputResW">
           <property name="accessibleName">
            <string/>
           </property>
           <attribute name="title">
            <string>输出结果</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPlainTextEdit" name="outputResPlain">
              <property name="styleSheet">
               <string notr="true">QPlainTextEdit#outputResPlain{
	border:0;
}</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="CompResW">
           <attribute name="title">
            <string>统计结果</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableWidget" name="recordResTable">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QTableWidget#recordResTable{
	border:0;
}</string>
              </property>
              <column>
               <property name="text">
                <string>种群大小</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>交叉概率</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>变异概率</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>迭代次数</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>最优适应度</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>最优结果</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>运行时间</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="para_Set">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>300</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>参数设置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>12</number>
           </property>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>变异概率:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>迭代次数:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="var_Spin">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="keyboardTracking">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <double>0.010000000000000</double>
             </property>
             <property name="maximum">
              <double>0.100000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>交叉慨率:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="popu_Spin">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>10</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>种群大小:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="cross_Spin">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="keyboardTracking">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <double>0.600000000000000</double>
             </property>
             <property name="maximum">
              <double>0.900000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.050000000000000</double>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QGroupBox" name="selectAlgo_GB">
             <property name="title">
              <string>选择算法</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <widget class="QRadioButton" name="wheel_Btn">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>20</y>
                <width>118</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>轮盘</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
             <widget class="QRadioButton" name="prize_Btn">
              <property name="geometry">
               <rect>
                <x>140</x>
                <y>20</y>
                <width>118</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>二元锦标</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QPushButton" name="return_ProMain">
             <property name="text">
              <string>返回</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="startAlgo">
             <property name="text">
              <string>开始</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="iterNums_Spin">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="keyboardTracking">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <number>50</number>
             </property>
             <property name="maximum">
              <number>400</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
